src/status_im/contexts/communities/discover/view.cljs

```diff
@@ -132,32 +131,29 @@
    [discover-communities-segments selected-tab false]])
 
 (defn other-communities-list
-  [{:keys [communities communities-ids view-type]}]
+  [{:keys [communities view-type]}]
   [rn/view {:style style/other-communities-container}
-   (if (and communities (pos? (count communities)))
-     (map-indexed
-      (fn [inner-index item]
-        (let [community-id (when communities-ids item)
-              community    (if communities
-                             item
-                             (rf/sub [:communities/home-item community-id]))
-              cover        {:uri (get-in (:images item) [:banner :uri])}]
-          [rn/view
-           {:key           (str inner-index (:id community))
-            :margin-bottom 16}
-           (if (= view-type :card-view)
-             [quo/community-card-view-item
-              {:community (assoc community :cover cover)
-               :on-press  #(rf/dispatch [:communities/navigate-to-community-overview (:id community)])}]
-
-             [quo/community-list
-              {:on-press      (fn []
-                                (rf/dispatch [:dismiss-keyboard])
-                                (rf/dispatch [:communities/navigate-to-community-overview
-                                              (:id community)]))
-               :on-long-press #(js/alert "TODO: to be implemented")}
-              community])]))
-      (if communities communities communities-ids))
```

Comment:
Lots of issues in this function, `communities-ids` was never passed and the logicis not even prepared to use it if passed

Reply:
Hmm odd that with what's happening with `communities-ids` in the original version. And it seems as you say there's no other usages of this function.

Nice clean up, thank you! 🙌